spec/Candid.md

```diff
@@ -1001,7 +1001,7 @@ The relations above have certain properties. As in the previous section, `<v> :
 
 * Roundtripping:
   ```
-  (<v> : <t>) ⟺ C[<t> <: <t>](<v>) = <v>
+  (<v> : <t>) ⟺ C[<t> ~> <t>](<v>) = <v>
   ```
 
 * Soundness of subtyping (or, alternatively, well-definedness of coercion)
```

Comment:
This needs to be now split into two properties:

* Well-typedness:
  ```
  ∀ <v> : <t>. C[<t> <: <t'>](<v>) : <t'> (if defined)
  ```
* Totality (on subtypes):
  ```
  <t> <: <t'> ⇒ ∀ <v> : <t> C[<t> <: <t'>](<v>) is defined
  ```